build.gradle

```diff
@@ -4,14 +4,14 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.3'
-        classpath 'org.kt3k.gradle.plugin:coveralls-gradle-plugin:2.8.2'
+        classpath "com.android.tools.build:gradle:3.5.2"
+        classpath "org.kt3k.gradle.plugin:coveralls-gradle-plugin:2.8.2"
         classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
     }
 }
 
 plugins {
-    id 'com.github.ben-manes.versions' version '0.20.0'
+    id "com.github.ben-manes.versions" version "0.27.0"
```

Comment:
fyi - i think we can get rid of this now that Gradle/Android Studio comes with built-in version checking

Reply:
Yeah I've had mixed results with that. It works generally, but also sometimes misses dependencies that have not been fetched already by other projects or manually via this plugin.